Gemfile.lock

```diff
@@ -1,62 +1,60 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    better_errors (2.1.1)
+    better_errors (2.5.0)
```

Comment:
### [better_errors](https://github.com/BetterErrors/better_errors)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 2.1.1 → 2.5.0

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **135** commits. See the full changes on [the compare page](https://github.com/BetterErrors/better_errors/compare/v2.1.1...v2.5.0).

These are the first 10 commits:
- (c0cda1e) [Test with Ruby 2.2 version](https://github.com/BetterErrors/better_errors/commit/c0cda1ef95c72cb203af86cc7333e8b95766568d)
- (b5de722) [Merge pull request #299 from nicolasleger/patch-1](https://github.com/BetterErrors/better_errors/commit/b5de722fb4ff813192f6268659835df7f6a52761)
- (d22d8ec) [Add section about text editor settings](https://github.com/BetterErrors/better_errors/commit/d22d8ec29afd2f77b7df992fb58abf48739827f8)
- (c57231c) [fixed typo in editor protocol spec](https://github.com/BetterErrors/better_errors/commit/c57231c0345be1654245f63de00d502684fd2c39)
- (e3fddb9) [Merge pull request #317 from amPerl/master](https://github.com/BetterErrors/better_errors/commit/e3fddb92800eb4a28baeb87d17ca92f296ea9656)
- (4b146fc) [add idea + rubymine app url capability](https://github.com/BetterErrors/better_errors/commit/4b146fcef6d6359d65a5013493af61355d326b3c)
- (a65b56d) [Consolidate newlines in exception messages to avoid blank ou…](https://github.com/BetterErrors/better_errors/commit/a65b56d03e0cb4a16ad02775631238068256b419)
- (a52bbdf) [Merge pull request #323 from latortuga/strip-leading-space](https://github.com/BetterErrors/better_errors/commit/a52bbdf1d969c33a71f7365c4919f1019f359649)
- (a05a894) [ignore special #$! local variable in StackFrame#local_variab…](https://github.com/BetterErrors/better_errors/commit/a05a894134d62002e88f1e951beca90d2ddd06fa)
- (748cb59) [Fix issues with pry version >= 0.10](https://github.com/BetterErrors/better_errors/commit/748cb59a72d72645be9637cdb2298f7e2358400b)


</details>